pypuf/learner/regression/logistic_regression.py

```diff
@@ -227,7 +226,4 @@ def learn(self):
             distance = (self.training_set.N - count_nonzero(self.training_set.responses == self.sign_combined_model_responses)) / self.training_set.N
             self.min_distance = min(distance, self.min_distance)
 
-        if not converged and distance > .01:
-            stderr.write('\rNOT CONVERGED                    ')
```

Comment:
This is an important log message that -- I believe -- is not yet implemented in the new code. But we need it! :)